src/entity/Address.ts

```diff
@@ -0,0 +1,32 @@
+import { Entity, PrimaryGeneratedColumn, Column, ManyToOne } from 'typeorm';
+import { User } from './User';
+
+@Entity()
+export class Address {
+  @ManyToOne(() => User, (user) => user.addresses)
+  user: User;
+
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column()
+  cep: string;
+
+  @Column()
+  street: string;
+
+  @Column()
+  streetNumber: string;
+
+  @Column({ nullable: true })
+  complement: string;
```

Comment:
```suggestion
  complement?: string;
```
a anotação avisa o banco que a coluna no campo é opcional, mas o typescript não infere que é opcional, por isso o `?`